common/source/docs/common-esc-telem-based-notch.rst

```diff
@@ -16,9 +16,7 @@ If ESCs with telemetry are used (either via a separate telemety wire or bi-direc
 - Set :ref:`INS_HNTCH_FREQ <INS_HNTCH_FREQ>` and/or :ref:`INS_HNTC2_FREQ <INS_HNTC2_FREQ>` = below the hover frequency - you can easily determine this by performing a gentle hover and looking at the ESC telemetry data
 - Set :ref:`INS_HNTCH_BW <INS_HNTCH_BW>` and/or :ref:`INS_HNTC2_BW <INS_HNTC2_BW>` = half of INS_HNTCH_FREQ
 
-.. note::
-
-   If you set the multi-source option (i.e. a notch per motor) mentioned below, the bandwidth should not be half the frequency. It should be greatly reduced as more notches cause more phase lag (i.e. latency), thus the bandwidth needs to be reduced to maintain a reasonable phase lag, else oscillation and a poorer tune will result despite the more accurate filtering compared to throttle-based filtering. The suggested starting point is :ref:`INS_HNTCH_FREQ<INS_HNTCH_FREQ>` / number of motors. For example, with a quadcopter, :ref:`INS_HNTCH_FREQ<INS_HNTCH_FREQ>` / 4 is suggested as a starting point.
+.. warning:: If you set the multi-source option (i.e. a notch per motor) mentioned below, the bandwidth should not be half the frequency. It should be greatly reduced as more notches cause more phase lag (i.e. latency), thus the bandwidth needs to be reduced to maintain a reasonable phase lag, else oscillation and a poorer tune will result despite the more accurate filtering compared to throttle-based filtering. The suggested starting point is :ref:`INS_HNTCH_BW<INS_HNTCH_BW>` / number of motors. For example, with a quadcopter, :ref:`INS_HNTCH_BW<INS_HNTCH_BW>` / 4 is suggested as a starting point.
```

Comment:
It should be INS_HNTCH_FREQ/4 we have already proved that F/number of motors is in fact wrong in a bad way.

Reply:
this should be better now, hopefully

Reply:
I think just take out :ref:`INS_HNTCH_BW<INS_HNTCH_BW>` / number of motors - just say starting point should be :ref:`INS_HNTCH_BW<INS_HNTCH_BW>` / 4 - that's always a good starting point and the former is wrong in many instances

Reply:
yep...fixed it on esc and forgot to copy it to fft...